parameter.go

```diff
@@ -31,17 +31,29 @@ func NewParameters(basePath string, parameters map[string]*Parameter) *Parameter
 
 //Parameters holds the output and all AWS Parameter Store that have the same base path
 type Parameters struct {
+	i          int64 // read index
```

Comment:
Please rename `i` to something more readable:
```suggestion
	readIndex          int64 
```